katas/content/distinguishing_unitaries/index.md

```diff
@@ -128,6 +128,46 @@ This kata offers you a series of tasks in which you are given one unitary from t
     "title": "Distinguishing Multi-Qubit Gates"
 })
 
+@[exercise]({
+    "id": "distinguishing_unitaries__ix_cnot",
+    "title": "IX or CNOT",
```

Comment:
We need to clarify in the title that it's not a product of I and X gates but their tensor product. A Unicode symbol should work fine:
```suggestion
    "title": "I⊗X or CNOT?",
```